assets/js/googlesitekit/widgets/index.js

```diff
@@ -43,6 +43,27 @@ const Widgets = {
 	 */
 	components: { Widget },
 
+	/**
+	 * Default contexts for Site Kit widgets.
+	 *
+	 * @since n.e.x.t
+	 */
+	contexts,
+
+	/**
+	 * Default areas for Site Kit widgets.
+	 *
+	 * @since n.e.x.t
+	 */
+	areas,
+
+	/**
+	 * Supported widths for Site Kit widgets.
+	 *
+	 * @since n.e.x.t
+	 */
+	widths: WIDGET_WIDTHS,
```

Comment:
Let's expose this one at the same capitalized name `WIDGET_WIDTHS`. Also, while we're at it, we should expose `WIDGET_STYLES` here as well. Contrary to the default contexts and areas, these are constant lists which Site Kit fully controls, so it's more appropriate to expose here IMO.

Reply:
Updated.